docs/source/deployment.rst

```diff
@@ -53,3 +53,9 @@ Config apache
 -------------
 
 replace ``PDC_HOSTNAME`` with server's hostname in ``/etc/httpd/conf.d/pdc.conf``
+
+
+Enabled proxy
+-------------
+
+Add ``USE_X_FORWARDED_HOST = True`` in `local_setting.py` file.
```

Comment:
I'd add a 1-2 sentences explaining why it has to be done. Something like - "To make sure documentation links work correctly when PDC is running behind proxy, add ...."